libkbfs/interfaces.go

```diff
@@ -1687,9 +1688,6 @@ type BareRootMetadata interface {
 	// GetCurrentTLFPublicKey returns the TLF public key for the
 	// current key generation.
 	GetCurrentTLFPublicKey(ExtraMetadata) (kbfscrypto.TLFPublicKey, error)
-	// AreKeyGenerationsEqual returns true if all key generations in the passed metadata are equal to those
-	// in this revision.
-	AreKeyGenerationsEqual(kbfscodec.Codec, BareRootMetadata) (bool, error)
```

Comment:
This is used by the server to detect when new MDs are rekeyed in Put. But this can be done by just comparing the output of `GetUserDevicePublicKeys`.